visualization/app/codeCharta/ui/fileChooser/fileChooser.component.spec.ts

```diff
@@ -86,28 +86,28 @@ describe("fileChooserController", () => {
 				size: 45
 			} as File
 
-			fileChooserController["addNameDataPair"](file0, '{"fileChecksum":"invalid-but-present-md5-checksum"}')
+			fileChooserController["addNameDataPair"](file0, '{"fileChecksum":"invalid-but-present-md5-checksum"}', 0)
```

Comment:
Nit: I would add a new variable `const fileIndex = 0` and use that in the `expect()` calls as well. It's not blocking though.